indico/modules/events/abstracts/settings.py

```diff
@@ -37,6 +37,24 @@ class BOACorrespondingAuthorType(RichEnum):
     speakers = 'speakers'
 
 
+class BOALinkFormatting(RichEnum):
+    '''LaTeX book of abstracts link format setting
+
+    value is a 2-tuple of strings:
+    first is the hyperref option to use
+    second sets additional tex commands
+
+    See `indico/legacy/webinterface/tpls/latex/inc/document.tpl`
```

Comment:
rst uses dual-backticks to wrap monospaced text, but I don't think the reference is needed. It's something that'll likely get outdated at some point ;)